src/main/java/seedu/address/model/tasks/exceptions/TaskNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.tasks.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified task.
+ */
+public class TaskNotFoundException extends RuntimeException {
```

Comment:
For now, this is thrown when we are trying to delete tasks, similar to when we are trying to delete a contact from the contact list.

If a find task is to be implemented, it would follow how `find` is implemented, ie through predicates and if none matches, no tasks will show up instead of throwing an exception